src/public/publicAPIs.ts

```diff
@@ -12,20 +12,18 @@ import { logs } from '../private/logs';
  * Initializes the library. This must be called before any other SDK calls
  * but after the frame is loaded successfully.
  */
-export function initialize(hostWindow: any = window): void {
+export function initialize(callback?: () => void): void {
```

Comment:
Do we want to support passing output values of initialize call to callback (context: string, clientType: string) ?

Reply:
I don’t think so. The frameContext is an internal detail of the SDK used to determine what APIs are supported. We’ve never exposed that to developers. If we ever need to, we would do so via the getContext API.

The hostClientType is already available via the getContext API and I think that adding a parallel option for fetching it would be confusing.